src/references/rainbow-token-list/index.ts

```diff
@@ -65,32 +69,27 @@ function generateDerivedData(tokenListData: TokenListData) {
   return derivedData;
 }
 
-async function readRNFSJsonData<T>(filename: string): Promise<T | null> {
+function readMMKVJsonData<T>(filename: string): T | null {
```

Comment:
not sure if we need those wrappers at all. They are not doing anything useful now? Maybe we can just call directly mmkv?

Reply:
It handles the JSON parse errors so I would keep